addons/medical_ai/XEH_postInit.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+
+["ace_settingsInitialized", {
+    // Only run for AI that does not have to deal with advanced medical
+    if (EGVAR(medical,enableFor) == 1 || {hasInterface && {EGVAR(medical,level) == 2}}) exitWith {};
+
+    ["ace_firedNonPlayer", {
+        _unit setVariable [QGVAR(lastFired), CBA_missionTime];
+    }] call CBA_fnc_addEventHandler;
+
+    if (hasInterface) then {
+        ["ace_unconscious", {
+            params ["_unit", "_unconscious"];
+            if (!_unconscious || {_unit != ACE_player}) exitWith {};
+
+            private _medic = objNull;
+            {
+                if ((!isPlayer _x) && {[_x] call EFUNC(medical,isMedic)}) exitWith {
+                    _medic = _x;
+                };
+            } forEach (units _unit);
+            if (isNull _medic) exitWith {};
+
+            private _healQueue = _medic getVariable [QGVAR(healQueue), []];
```

Comment:
Doesn't matter, the medic checks before moving to the unit if it still needs healing.